### ISSUE TITLE
SBT cannot find the package

### ISSUE THREAD

**Issue by vasily-kirichenko** [dnvriend/akka-persistence-inmemory#60]
`build.sbt`

```scala
scalaVersion := "2.12.7"
val akka_version = "2.5.17"

resolvers ++= Seq(
    "jitpack" at "https://jitpack.io",
    Resolver.jcenterRepo
)

libraryDependencies ++= Seq(
    "com.typesafe.akka" %% "akka-actor-typed" % akka_version,
    "com.typesafe.akka" %% "akka-stream-typed" % akka_version,
    "com.typesafe.akka" %% "akka-persistence-typed" % akka_version,
    "com.typesafe.akka" %% "akka-persistence-query" % akka_version,
    "com.typesafe.akka" %% "akka-cluster-sharding-typed" % akka_version,
    "com.typesafe.akka" %% "akka-cluster-typed" % akka_version,
    "com.typesafe.akka" %% "akka-actor-testkit-typed" % akka_version % Test,
    "com.typesafe.akka" %% "akka-stream-testkit" % akka_version % Test,
    "com.typesafe.akka" %% "akka-http" % "10.1.5",
    "com.typesafe.akka" %% "akka-slf4j" % akka_version,
    "com.typesafe.akka" %% "akka-distributed-data" % akka_version,
    "com.github.dnvriend" %% "akka-persistence-jdbc" % "3.4.0",

    "com.github.dnvriend" % "akka-persistence-inmemory_2.11" % "2.5.15.1" % Test

    // some other deps here
)
```
Result:
```
[warn] ==== jitpack: tried
[warn]   https://jitpack.io/com/github/dnvriend/akka-persistence-inmemory_2.11/2.5.15.1/akka-persistence-inmemory_2.11-2.5.15.1.pom
[warn] ==== jcenter: tried
[warn]   https://jcenter.bintray.com/com/github/dnvriend/akka-persistence-inmemory_2.11/2.5.15.1/akka-persistence-inmemory_2.11-2.5.15.1.pom
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	::          UNRESOLVED DEPENDENCIES         ::
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	:: com.github.dnvriend#akka-persistence-inmemory_2.11;2.5.15.1: not found
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn]
[warn] 	Note: Unresolved dependencies path:
[warn] 		com.github.dnvriend:akka-persistence-inmemory_2.11:2.5.15.1 (C:\WLCR\google-play-crawler\build.sbt#L19-51)
[warn] 		  +- default:google-play-crawler_2.12:0.1
[error] sbt.librarymanagement.ResolveException: unresolved dependency: com.github.dnvriend#akka-persistence-inmemory_2.11;2.5.15.1: not found
```

**Comment by alexmillerfairsailcom** [comment]
The latest version available on Maven Central or jCenter is 2.5.1.1

**Comment by vasily-kirichenko** [comment]
As have answered on SO, see https://stackoverflow.com/questions/52963362/sbt-cannot-find-a-package-on-jcenter, the latest version is available on Bintray:
```scala
resolvers += "dnvriend" at "http://dl.bintray.com/dnvriend/maven"
```
It does solved the issue.
